.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+language: android
+jdk: oraclejdk8
+android:
+ components:
+    - platform-tools
+    - tools
+    - build-tools-26.0.2
+    - android-26
+    - extra-android-m2repository
+before_script:
+ - chmod +x gradlew
+script:
+- ./gradlew build --stacktrace
```

Comment:
the build is failing because of the lint issues. 
Do one of the following
1. either send a PR to fix lint issues
2. disable lint check in the build.gradle
3. replace `./gradlew build` by `./gradlew assembleDebug`

Reply:
Shall I update the PR with disabling lint check (so that  travis build doesn't fail) & create another PR with solving lint issues (will enable the lint issues in this PR) ?

Reply:
It's better if to use assembleDebug.
Also add a todo that it's to be changed once we fix lint issues